tests/gnmi/test_gnmi_configdb.py

```diff
@@ -69,6 +69,8 @@ def test_gnmi_configdb_incremental_01(duthosts, rand_one_dut_hostname, ptfhost):
     Toggle interface admin status
     '''
     duthost = duthosts[rand_one_dut_hostname]
+    if duthost.is_supervisor_node():
```

Comment:
hi @arista-nwolfe , in the case when sup should not run the test, we recommend to use `enum_rand_one_frontend_node` instead of `rand_one_dut_hostname`, the former fixture will ensure selected dut is not sup

Reply:
nvm, since the test file and the module's conftest's fixture needs to be same, we have to use `rand_one_dut_hostname` here..
Hence this is the only solution
LGTM